tests/tests/regional_2threads

```diff
@@ -46,3 +46,7 @@ NTILES=1
 
 ESMF_THREADING=false
 THRD=$atm_omp_num_threads
+
+if [[ ${MACHINE_ID} = ursa ]]; then
+   WLCLK=30
+if
```

Comment:
@jkbk2004 @BrianCurtis-NOAA This is incorrect and the RTs fail on Hera. Should be `fi`, not `if`